### DISCUSSION TITLE
Consider supporting extended colours for markers

### DISCUSSION THREAD

**Discussion by lloydjatkinson** [expressive-code/expressive-code#377] [Feature Requests]
Taken from [here](https://rehype-pretty.pages.dev/#group-highlighted-chars-by-id).
![image](https://github.com/user-attachments/assets/d5e5afff-1ae8-4b8e-8f31-df482bb3d080)

It would be really cool if markers could somehow support syntax for extra colours. `{"1. Provide the value prop here:":5-6}` or `js "given text"` - I'm just not sure what that would look like.

**Comment by delucis** [comment]
Hello! We are moving feature requests and support questions to discussions and using issues only for bugs to help better track those going forward. See you in the [Discussions](https://github.com/expressive-code/expressive-code/discussions) tab!
